cpptest.lua

```diff
@@ -0,0 +1,19 @@
+project "cpptest"
+
+dofile(_BUILD_DIR .. "/static_library.lua")
+
+configuration { "*" }
+
+uuid "394726C9-4069-4CE2-8BE4-A8C4281B833F"
+
+includedirs {
+  "clang", -- for cpptest config.h (for clang)
+  "win",   -- for cpptest winconfig.h (for msvc)
+}
+
+files {
+  "src/**.h",
+  "src/**.cpp",
+  "clang/config.h",
+  "win/winconfig.h",
+}
```

Comment:
Should the desktop platform booleans be added here or is this truly global for Linux, macOS and Windows?

Reply:
I'm not sure it matters. Here's the pattern used for including these "config" headers in the code:
```cpp
#if (defined(__WIN32__) || defined(WIN32))
# include "winconfig.h"
#else
# include "config.h"
#endif 
```
E.g. [here](https://github.com/Esri/cpptest/blob/master/src/collectoroutput.cpp#L27-L31)

Given that the conditionalization is done via `#ifdef` in the source, to conditionalize the headers' visibility and presence in solution files seems a little redundant. It's just a single, small file: it seems like it would be nice to be able to see `config.h` in visual studio even if you never include it.